File: tests/rest-assured-java/src/test/java/com/portfolio/qa/endpoints/Endpoints.java
package com.portfolio.qa.endpoints;

public final class Endpoints {

    private Endpoints() {
    }

    public static final String HEALTH = "/health";

    public static final String TASKS = "/api/v1/tasks";
    public static final String TASK_BY_ID = "/api/v1/tasks/{id}";
    public static final String TASK_STATUS_BY_ID = "/api/v1/tasks/{id}/status";

    public static final String TEST_RESET = "/api/v1/test/reset";
}

File: tests/cypress-ts/src/core/endpoints.ts
export const Endpoints = {
  health: '/health',
  docs: '/docs/',
  testReset: '/api/v1/test/reset',
  tasks: '/api/v1/tasks',

  taskById: (taskId: number): string =>
    `/api/v1/tasks/${taskId}`,

  taskStatusById: (taskId: number): string =>
    `/api/v1/tasks/${taskId}/status`
} as const;